sessionctx/context.go

```diff
@@ -182,6 +182,12 @@ type Context interface {
 	ReleaseAllAdvisoryLocks() int
 	// GetExtensions returns the `*extension.SessionExtensions` object
 	GetExtensions() *extension.SessionExtensions
+	// InSandBoxMode indicates that this Session is in sandbox mode
+	InSandBoxMode() bool
+	// EnableSandBoxMode enable the sandbox mode of this Session
+	EnableSandBoxMode()
```

Comment:
I don't get what is a sandbox mode at the first sight. Explain more, and probably rename it to something like `ExpiredPasswordMode` or so.

Reply:
`Sandbox Mode` is a [term](https://dev.mysql.com/doc/refman/8.0/en/expired-password-handling.html) from mysql. `InSandBoxMode` indicates that current session can only execute `set password` and `alter user` statements to reset the password.

Reply:
Then maybe reference the link to the code instead of in the issue only.